packages/flutter_tools/test/commands.shard/hermetic/upgrade_test.dart

```diff
@@ -40,6 +41,9 @@ void main() {
   });
 
   testUsingContext('can auto-migrate a user from dev to beta', () async {
+    // Ensure the the cache root exists so caching will work.
+    await globals.cache.getRoot().create(recursive: true);
```

Comment:
nit: we usually just use the sync APIs, especially in tests

Reply:
Done.